src/main/java/school/lemon/changerequest/java/jdbc/Demo.java

```diff
@@ -0,0 +1,33 @@
+package school.lemon.changerequest.java.jdbc;
```

Comment:
Please read [requirements for Demo](https://github.com/ChangeRequest/jdbc.hw1/blob/master/README.md#4-demo-class) class and implement class in a way that:

> Create demo class, that creates:
> 
> 1. Several items.
> 2. Several categories.
> 3. Several catalogs.
> 4. Several properties.
> 5. Link items to catalogs, categories, properties.
> 6. Retrieve information about all catalogs, with all items in this catalog. Note that properties and categories of all items should be filled in.

You **have to** execute SQL statements from Demo class, not from raw `.sql` files.